lib/stat_tracker.rb

```diff
@@ -80,4 +81,14 @@ def biggest_bust(season_param)
     season = SeasonStat.new(@games_file, @teams_file, @game_teams_file)
     season.biggest_bust(season_param)
   end
+
+  def biggest_surprise(season_param)
+    season = SeasonStat.new(@games_file, @teams_file, @game_teams_file)
+    season.biggest_surprise(season_param)
+  end
+
```

Comment:
this pull request also has biggest surprise, thought I PRed that last night, sorry guys!